rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+
+    @book = book
+    book.rental << self
+
+    @person = person
+    person.rental << self
+  end
+end
```

Comment:
- [ ] Currently the user is unable to list rentals using `id`, kindly fix this as it is a requirement
![image](https://user-images.githubusercontent.com/81932478/210333343-426f6b99-c607-449f-af62-da37f143cad9.png)

Reply:
@Munsa1 The person ID is not 1. You can see the person ID if you display all persons.